.github/workflows/benchmark.yml

```diff
@@ -60,7 +60,15 @@ jobs:
         from-lock: 'true'
 
     - name: Run benchmarks
-      run: uv run pytest --db-backend psql --benchmark-only --benchmark-json benchmark.json tests/
+      run: pytest --db-backend psql --benchmark-only --benchmark-json benchmark.json tests/test_conftest.py
+
+    - name: Check if MY_ENV_VAR is set
+      run: |
+        if [ -z "${GITHUB_TOKEN}" ]; then
```

Comment:
Can You try to print the `${{ gitHub.repository }}`. According to the condition on line 22 two this workflow should not run on forks, yet somehow it does.

Reply:
ah right, I thought because I have maintainer rights I still would access the secrets from the aiidateam repo and that the problem is related to uv but that did not make actually sense. I made a PR with a potential fix https://github.com/aiidateam/aiida-core/pull/6894 Could you have a look?